client/my-sites/promote-post/utils/index.ts

```diff
@@ -118,8 +118,14 @@ export const getCampaignDurationFormatted = ( start_date: string, end_date: stri
 	return durationFormatted;
 };
 
-export const getCampaignBudgetData = ( budget_cents: number, spent_budget_cents: number ) => {
-	const totalBudget = budget_cents / 100;
+export const getCampaignBudgetData = (
+	budget_cents: number,
+	start_date: string,
+	end_date: string,
+	spent_budget_cents: number
+) => {
+	const totalBudget =
+		( budget_cents * moment( end_date ).diff( moment( start_date ), 'days' ) ) / 100;
```

Comment:
@benzhu56 One thing worth noting if you weren't already aware.   Is that the moment team, no longer recommend using moment. Probably used all over in calyspo and a huge task to remove totally.    But in general I try to avoid using it now where I can, and never add to new projects - package size is huuuge.

https://momentjs.com/docs/#/-project-status/

Reply:
Thanks for mentioning this! I'll use an alternative.